kolibri/core/logger/tasks.py

```diff
@@ -116,3 +169,18 @@ def exportsummarylogcsv(facility_id, **kwargs):
         kwargs.get("end_date"),
         kwargs.get("locale"),
     )
+
+
+@register_task()
```

Comment:
It would be useful to add in a job ID to this task. The `register_task` decorator takes the `job_id` as an argument, which will always be used when the task is registered. Using this predictable id deduplicates it and will ensure that there is only one cleanup task running at once.